### DISCUSSION TITLE
Thickness modifier for extrude?

### DISCUSSION THREAD

**Discussion by LaineZ** [dune3d/dune3d#142] [Q&A] [answered]
Hello. I am wondering if there is a function for extruded bodies like shell/enclousure operation from KOMPAS-3D or thickness task from FreeCad?

Screenshot from freecad:

![image](https://github.com/user-attachments/assets/01ec40fe-22a9-45bb-8b2d-42fd2defa485)

**Comment by carrotIndustries** [comment, marked as answer]
So far no.
